blog/_posts/2021-08-20-from-c++-to-js.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "The Fast & Furious Data Visualizations"
+date: August 20, 2021
+author: David E. Ackley
+---
+
+# The Fast & Furious Data Visualizations
+#### August 20, 2021 David E. Ackley
+
+The main data visualizations tools from the D3 wrapper that was created by ...
+This project is a part of Empirical, a header-only C++ library equipped with tools for developing experimental software.
+
+
+## A brief history lesson on Javascript
+invented by Brendan Eich
+
+
+### The bias against JavaScript
+
+
+### ECMAScript versions over the years
+
+## Turbocharging the Javascript Engine
+### compiling c++ to wasm
+### spidermonkey and v8
+
+## Running on undefined
```

Comment:
I would start writing here (or a bit further down where you get into the d3 wrapper meat of what you’ve been focusing on) and then as time (and space) allows fill in the sections above